docker/debian/Dockerfile

```diff
@@ -28,7 +28,11 @@ ENV MINIFI_BASE_DIR /opt/minifi
 ENV MINIFI_HOME $MINIFI_BASE_DIR/nifi-minifi-cpp-$MINIFI_VERSION
 ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64/
 
-RUN apt-get update && apt-get install -y openjdk-8-jdk libpython3.5-dev openjdk-8-source sudo git maven
+RUN echo "deb http://deb.debian.org/debian stretch-backports main" | tee -a /etc/apt/sources.list \
+    && echo "deb http://deb.debian.org/debian stretch-backports-sloppy main" | tee -a /etc/apt/sources.list \
+    && apt-get update && apt-get install -y openjdk-8-jdk libpython3.5-dev openjdk-8-source sudo git maven \
+    && apt-get -t stretch-backports-sloppy install -y libarchive13\
```

Comment:
Why do we need the system libarchive?  We bundle that.

Reply:
It's a dependency of cmake. The maintainer incorrectly uploaded cmake 3.16 (instead of 3.13) to the amd64 stretch-backports, which depends on libarchive >=3.3.3. The version in the stretch repos is 3.2 and there is no libarchive in stretch-backports, so this is a workaround. 

See this bug: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=954852
and see the inconsistent version near the bottom of this page: https://packages.debian.org/stretch-backports/cmake